### DISCUSSION TITLE
Email automations [Aug 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3045] [Discord Threads]
> 🙋‍♂️ **𝘿𝙚𝙛𝙖𝙘𝙚𝙙𝘿𝙖𝙩𝙖**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### NocoDB has webhooks, but CRON/Timer based triggers are currently unavailable  
#  
> 🙋‍♂️ **rallisf1**  
#### I am using emailengine.app with webhooks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1002686551731486770)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
